CMakeLists.txt

```diff
@@ -115,8 +115,8 @@ src/Proto/SecurityCommandBuffer.pb.cc
 src/Proto/box2d.pb.cc
 src/Shared/Timer.cpp
 src/Shared/fake_cpp11.hpp
-src/Application/Configuration.cpp
-src/Application/Configuration.h
+src/Application/Config.cpp
```

Comment:
I figured since we are going to be referencing the config instance a lot it could lead to some pretty long lines of code, especially when passed as function arguments.  So I thought making it short and conscise would reduce the amount of line wrapping and would be easier to read.  However this can be fixed by storing the instance in a short named local variable first.  So I changed it back to the long form everywhere.